lib/config/flow.js

```diff
@@ -12,12 +12,7 @@ module.exports = {
     sourceType: 'module',
   },
 
-  plugins: [
-    'flowtype',
-    'shopify',
-  ],
+  plugins: ['flowtype', 'shopify'],
```

Comment:
There's also a config for `prettier/flowtype` (in addition to one for React) — do we want those?

Reply:
Yup. That work can be in a future PR. We'll have to manually find out which rules from those configs we'll need to turn off.